client/blocks/comment-detail/comment-detail-author.jsx

```diff
@@ -164,4 +201,12 @@ export class CommentDetailAuthor extends Component {
 	}
 }
 
-export default localize( CommentDetailAuthor );
+const mapStateToProps = ( state, ownProps ) => {
+	const settings = getSiteSettings( state, ownProps.siteId );
+	return {
+		dateFormat: get( settings, 'date_format' ),
+		timeFormat: get( settings, 'time_format' ),
```

Comment:
For example instead of using a util from another folder, we can encapsulate logic in `phpToMomentDatetimeFormat` and `getFormattedDate` to something like `getHumanPostDate`